core-processor/src/common.rs

```diff
@@ -357,15 +378,22 @@ pub struct Actor {
 pub struct ExecutableActorData {
     /// Program.
     pub program: Program,
-    /// Data which some program allocated pages may have.
-    pub pages_data: BTreeMap<PageNumber, PageBuf>,
+    /// Numbers of memory pages with some data.
```

Comment:
This formulae is really strange for me.
Can we replace it with something like: "allocated pages, that have non-default data"?